actionpack/lib/action_controller/metal/rendering.rb

```diff
@@ -54,6 +54,12 @@ def render_to_body(options = {})
 
     private
 
+    def _process_variant(options)
+      if (variant = request.variant)
```

Comment:
also changes a "nil or empty array?" check to a "nil?" check without explanation